pmdarima/compat/statsmodels.py

```diff
@@ -45,8 +45,18 @@ def check_seasonal_order(order):
     order : tuple
         The existing seasonal order
     """
-    if sum(order[:3]) == 0 and order[-1] == 1:
-        order = (0, 0, 0, 0)
+
+    try:
+        # Assume an iterable for order[0].
+        # If order[0] is then we don't perform check.
+        # issue#370: https://github.com/alkaline-ml/pmdarima/issues/370
+        for _ in order[0]:
```

Comment:
Is this an internal-only function? I ask because if not, I am not sure I am on board with this. Since a string is iterable in Python, if `order[0]` is a string, this would pass, right? Would it not be better to have something like:

```python
from collections.abc import Iterable

if isinstance(order[0], Iterable) and not isinstance(order[0], str):
    return order
else:
    if sum(order[:3]) == 0 and order[-1] == 1:
            order = (0, 0, 0, 0)
    
    return order
```

There is probably a more idiomatic way to write that, lol. On the other hand, I think this is just a pre-validation check, so if the user did pass something bad in, it would be caught by statsmodels

Reply:
Ahhh good point about catching string iterables.

Reply:
Done!